README.md

```diff
@@ -0,0 +1,6 @@
+HELLO WORLD PROJECT
+This is a simple hello world project for Module 1 Weekday 2
+
+For this project I used:
+HTML: to add an h1 tag containing hello world
+CSS: to style the background color of my h1 tag to my favorite color and also chnage the color of the text.
```

Comment:
- [ ] Please add a descriptive Readme to your project. Please use [this ](https://github.com/microverseinc/readme-template) template but remember to customize it to your project.